src/constants.ts

```diff
@@ -11,6 +11,7 @@ export const BREAKPOINTS = {
 export const DEFAULT_THEME = 'light';
 
 export const UIKIT_ROOT_CLASS = 'g-root';
+export const UIKIT_THEME_LIGHT_CLASS = 'g-root_theme_light';
```

Comment:
This is part of DEFAULT_THEME and part of UIKIT_ROOT_CLASS
I think there is no need for one more constant. Or you can use these two constants here

Reply:
i think about this solution, but it didn't seem so beautiful)

Reply:
fix